src/main/java/org/opentripplanner/routing/edgetype/SimpleTransfer.java

```diff
@@ -1,96 +1,44 @@
 package org.opentripplanner.routing.edgetype;
```

Comment:
Now that this is not an edge anymore, it is probably more correct to place it in the model package.

Reply:
Maybe we could rename it too, but I'm not sure what would it should be called.

Reply:
Moved, the question is if we should try to merge or move functionality to or from `org.opentripplanner.routing.algorithm.raptor.transit.Transfer` and/or `org.opentripplanner.routing.algorithm.raptor.transit.request.TransferWithDuration`. The only additional thing in the first is the RAPTOR stop index, which is not stable(?), and the latter one has the duration, which is not accurate, and can be wildly off, eg. if the transfer uses an elevator edge (relates to #2955).

Reply:
The Raptor stop index should be stable, and we could just have the Raptor stop index as a field on the Stop. This should probably be done as part of another pull request, when we clean up the TransitLayer classes and merge them with the OTP model classes where possible.